bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -2382,8 +2382,11 @@ void ReplicaImp::onTransferringCompleteImp(SeqNum newStateCheckpoint) {
 void ReplicaImp::onSeqNumIsSuperStable(SeqNum newSuperStableSeqNum) {
   auto seq_num_to_stop_at = controlStateManager_->getCheckpointToStopAt();
   if (seq_num_to_stop_at.has_value() && seq_num_to_stop_at.value() == newSuperStableSeqNum) {
-    if (getRequestsHandler()->getControlHandlers())
+    LOG_INFO(GL, "control state manager marked seqNum: " << newSuperStableSeqNum);
+    if (getRequestsHandler()->getControlHandlers()) {
+      metric_on_call_back_of_super_stable_cp_.Get().Set(1);
       getRequestsHandler()->getControlHandlers()->onSuperStableCheckpoint();
```

Comment:
The more I read `SuperStableCheckpoint`, the less I like it. This doesn't have to be done in this PR, as it will have ripple effects, but maybe we can change it to something like `NofNStableCheckpoint`. Then we can rename the callback to be `onNofNCheckpointsReceived`

I'm open to other naming ideas as well. If everyone disagrees, we can also keep it as is :)

Reply:
let's leach it for a future PR :)